simpleflu/README.md

```diff
@@ -56,46 +63,46 @@ Explicitly defining transition probabilities is one way of introducing stochasti
 Another way is to draw values from a probability distribution, demonstrated by defining the incubation period for each agent as a sample from a `lognormal` distribution
 
 ```fred
-state Exposed {
-    INFLUENZA.sus = 0
-    wait(24*lognormal(1.9,1.23))
-    next(InfectiousAsymptomatic) with prob(0.33)
-    default(InfectiousSymptomatic)
-}
+    state Exposed {
+        INFLUENZA.sus = 0
+        wait(24 * lognormal(1.9,1.23))
+        next(InfectiousAsymptomatic) with prob(0.33)
+        default(InfectiousSymptomatic)
+    }
 ```
 
 Both infectious states are identical with the exception of their transmissibility.
 Once entering one of the infectious states after the wait period in `Exposed`, agents gain a non-zero transmissibility and wait through an infectious period before recovering.
 Asymptomatic infections are half as transmissible as infectious ones, according to this model.
 
 ```fred
-state InfectiousSymptomatic {
-    INFLUENZA.trans = 1          # this value is .5 for InfectiousAsymptomatic
-    wait(24* lognormal(5.0,1.5))
-    next(Recovered)
-}
+    state InfectiousSymptomatic {
+        INFLUENZA.trans = 1          # this value is .5 for InfectiousAsymptomatic
+        wait(24* lognormal(5.0,1.5))
+        next(Recovered)
+    }
```

Comment:
I think we've agreed (for particular meanings of 'we' and 'agree') on tabs over spaces for indentation in FRED. This was only agreed after we wrote the tutorials, and I don't think it's documented anywhere. Only flagging it here as it appeared in the diff. We should probably formally agree on a style guide before modifying other code in the docs.

A nitpick and something that was only discussed after we wrote these

Reply:
I didn't even realize this.  That's just VS Code using tabs here by default and me blinding pasting this in.